crates/trycp_api/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "trycp_api"
+version = "0.0.100-alpha0"
```

Comment:
We should probably start versioning these, can it just match the tryorama minor version for now @jost-s?

Reply:
Cool